admin_guide/sdn_troubleshooting.adoc

```diff
@@ -495,6 +495,54 @@ normal operation, so do not be surprised to see entries there.
 
 Check external firewalls, if any, allow traffic to the target address (this is site-dependent, and beyond the purview of this guide).
 
+== Debugging Virtual Networking
```

Comment:
@adellape  I don't think putting in words "just because" is a good idea. Unless you think it's not clear what virtual network means?